snapshot_dbg_cli/snapshot_debugger_rtdb_service.py

```diff
@@ -154,38 +186,53 @@ def get_snapshot(self, debuggee_id, snapshot_id):
     if bp is None:
       bp = self.rest_service.get(snapshot_path)
 
-    return normalize_breakpoint(bp, snapshot_id)
+    return normalize_breakpoint(bp, breakpoint_id)
 
-  def get_active_breakpoints(self, debuggee_id, action, user_email):
-    path = self.schema.get_path_breakpoints_active(debuggee_id)
-    return self._get_breakpoints(path, action, user_email)
+  def get_snapshots(self, debuggee_id, include_inactive, user_email=None):
+    """Retrieves all the snapshots matching the search criteria.
 
-  def get_final_breakpoints(self, debuggee_id, action, user_email):
-    path = self.schema.get_path_breakpoints_final(debuggee_id)
-    return self._get_breakpoints(path, action, user_email)
+    To note, for any snapshots that have completed, no capture data is retrieved
+    via this call. To get the full capture data an individual call to
+    get_snapshot is required.
```

Comment:
Though appreciated, this comment is a bit odd.  It looks to me like either this method should be called "list_snapshots" or something that implies that it isn't deep, or get_snapshot should be renamed to something like "get_snapshot_detail" that implies that it *is* deep.

Not too critical though given that this is not user-facing.

Reply:
Good callout, I went with `get_snapshot_detailed`.